### PR TITLE
MB-14881-Customer-Gets-Clearer-Guidance-While-Completing-Steps-In-Their-PPM

### PR DESCRIPTION
**Pull request by dymonmoore**
## Summary

This pull request includes a series of changes to improve customer guidance throughout the PPM shipment process. 

**Expected Changes**
- When the customer views their review page, a departure date label should be displayed in the About Your PPM section
 
- When the customer views their entries for weight moved, user sees a Yes/No selection for if they’ve included a trailer in the trip 

-  When the customer views their entries for weight moved, a description label should be displayed

- When the customer views their entries for weight moved, a description label should be displayed before a pro-gear item

- When the customer has not uploaded any weight tickets for weight moved, the help text should say ‘**No weight moved documented. At least one trip is required to continue**.’ - See Duncan's comment below

- When the customer sees a default empty state for pro gear, they see ‘pro-gear’ rather than ‘spouse pro-gear’ when no selection is chosen

- When the customer views their entries for pro-gear items,  a description label should be displayed

- When the customer views their entries for expenses, a description and type label should be displayed

https://user-images.githubusercontent.com/111006369/216452861-4c9cd07d-7888-46e5-b18e-3689cf850970.mov

In this video, I will hover over new labels added. 

## Setup to Run Your Code

<details>
<summary>💻 You will need to use three separate terminals to test this locally.</summary>

##### Terminal 1

Start the Storybook locally.

```sh
make storybook
```

##### Terminal 2

Start the UI locally.

```sh
make client_run
```

##### Terminal 3

Start the Go server locally.

```sh
make server_run
```

</details>

## Verification Steps for Reviewers

These are to be checked by a reviewer.

1. Login in: readyToFinish@ppm.approved (milmove)
2. Select upload PPM documents
3. Check if 'Departure date' label is in About Your PPM section 
4. Check if 'Vehicle description' label is in weight moved section
5. Check if 'Trailer' is Yes/No
6. Add a new pro-gear weight item with a description, check for 'Description' label in display
7. Add a new expense, check fro 'Type' and 'Description' label
8. Add a new pro-gear weight item, do not add information, just save. Verify the user sees ‘pro-gear’ rather than ‘spouse pro-gear’ when no selection is chosen (default empty state)

## Screenshots
Label added are highlighted below: 
Screenshot 2023-02-03 at 9 49 33 AM

[MB-14903]: https://dp3.atlassian.net/browse/MB-14903?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by duncan-truss** [comment]
I was able to check for the empty weight tickets message by going directly to the review page URL for a PPM Shipment that didn't have any closeout data.  It's not too important though if you have a test and it will get confirmed later.

Screenshot 2023-02-03 at 11 27 25 AM

File: src/utils/ppmCloseout.jsx [hunk 1]
```diff
@@ -79,25 +82,32 @@ export const formatWeightTicketItems = (weightTickets, editPath, editParams, han
 
 export const formatProGearItems = (proGears, editPath, editParams, handleDelete) => {
   return proGears?.map((proGear, i) => {
-    const weightValues = proGear.hasWeightTickets
-      ? { id: 'weight', label: 'Weight:', value: formatWeight(proGear.weight) }
-      : { id: 'constructedWeight', label: 'Constructed weight:', value: formatWeight(proGear.weight) };
-    return {
```
Comment (duncan-truss) [inline src/utils/ppmCloseout.jsx:85]: I think we still want to display the Constructed weight label if the customer selects that they don't have weight tickets.

I think @dinabeigelman  possibly meant for the case where it was an empty pro-gear record to show the normal weight label as the default.
https://www.figma.com/file/tRkjtoWjT8727XH5m9k7mK/Milmove-Customer-App?node-id=3671%3A24753&t=rdQbBPW5kwB4saCq-4

Comment (dymonmoore) [reply src/utils/ppmCloseout.jsx:89]: I've added the constructed weight label per designs for empty weight tickets @duncan-truss @dinabeigelman

**Review by AnnaGingle — APPROVED** [review]
Look good!

**Review by duncan-truss — APPROVED** [review]

**Pull request merged**